### ISSUE TITLE
see more.. shown with < 5 values

### ISSUE THREAD

**Issue by grtjn** [marklogic/slush-marklogic-node#137]
if nr values smaller than facet limit, suppress see more..

**Comment by joemfb** [comment]
This should be on ml-search-ng, the functionality is totally contained there.

**Comment by grtjn** [comment]
Yes. Best to link to a ticket there, and keep this to update the bower dep..


Sent from Samsung Mobile

-------- Oorspronkelijk bericht --------
Van: Joseph Bryan <notifications@github.com> 
Datum: 03-09-2015  16:43  (GMT+01:00) 
Aan: marklogic/slush-marklogic-node <slush-marklogic-node@noreply.github.com> 
Cc: Geert <geert.josten@xs4all.nl> 
Onderwerp: Re: [slush-marklogic-node] see more.. shown with < 5 values (#137) 
 
This should be on ml-search-ng, the functionality is totally contained there.

**Comment by joemfb** [comment]
To be clear, you're asking that the stored search options be loaded initially, and the facet values be compared to their respective `facet-limit` property?

**Comment by grtjn** [comment]
Yeah, I guess that is how it needs to be done. Would be convenient if REST api would return a total facet count itself, but it doesn't. Although, we also have this includeAggregates that can provide a (total) count for a facet?

**Comment by hunterwilliams** [comment]
I realize this should be moved over to ml-search-ng but would the behavior be too odd if show more only appeared on facets with 5 or more values? Unfortunately 5 would look sorta weird, but it'd eliminate the awkwardness of 2-4 valued facets

**Comment by grtjn** [comment]
Crossed my mind too, but reluctant on a solution like that..

**Comment by grtjn** [comment]
If we can't find a solution for this in 0.2.0, I suggest we disable the show more for the moment. It looks confusing like it is now, despite the fact that it is a really nice feature..

**Comment by joemfb** [comment]
I don't think show-more needs to be on by default. It's trivial to add to a demo if requested.

**Comment by grtjn** [comment]
Pushing this into next milestone..
